myosuite/envs/myo/myochallenge/chasetag_v0.py

```diff
@@ -37,11 +37,13 @@ class ChallengeOpponent:
     Contains several different policies. For the final evaluation, an additional
     non-disclosed policy will be used.
     """
-    def __init__(self, sim, rng, probabilities: list, min_spawn_distance: float):
+    def __init__(self, sim, rng, probabilities: list, min_spawn_distance: float, linear_velocity: float):
```

Comment:
Let's make `linear_velocity` consistent with how other variables are passed ie. a tuple `(low, high)`. Depending on how the tuple is passed we can have none `(0,0)` , constant `(vel, vel)`, or random `(0, vel)`

Reply:
imlemented but didnt use `(0,0)` or negative velocities to distinguish between tasks, as we handle that through the `task_type='random'` argument

Reply:
This might need a close look. 
1. All function signatures must be changed to signify that it's a tuple.
2. It's still being compared as a string throughout. Comparision needs to move to tuple 
3. `task_choice` and `tuple` definition can conflict with each other. Over-specification might be a lot harder to handle.